data/experiment-recipe-samples/ios-39-rollout.json

```diff
@@ -0,0 +1,85 @@
+{
+  "appId": "org.mozilla.ios.Firefox",
+  "appName": "firefox_ios",
+  "application": "org.mozilla.ios.Firefox",
+  "arguments": {},
+  "branches": [
```

Comment:
This has multiple branches and affects multiple features. Should this be single branch and single feature?

Reply:
We would also like to test the interaction between rollouts and a multifeature experiment of this type, but that would have isRollout false.

Reply:
@heres-maria Can you compose the test data that would actually work for testing? We could add it to the repo after testing. I've tried a few times now in this PR - none of those tries has passed muster. Would rather just record what was used to successfully test the feature than keep guessing at what might work